Bleed compile issue

Looks like we overlooked some presumable rebase conflict:

```
Traits/Render/WithInfantryBody.cs(59,68): error CS1061: 'OwnerInit' does not contain a definition for 'PlayerName' and no accessible extension method 'PlayerName' accepting a first argument of type 'OwnerInit' could be found (are you missing a using directive or an assembly reference?) [/home/travis/build/OpenRA/OpenRA/OpenRA.Mods.Common/OpenRA.Mods.Common.csproj]
```

--- a/OpenRA.Mods.Common/Traits/Render/WithInfantryBody.cs
+++ b/OpenRA.Mods.Common/Traits/Render/WithInfantryBody.cs
@@ -56,7 +56,7 @@ namespace OpenRA.Mods.Common.Traits.Render
 			anim.PlayRepeating(RenderSprites.NormalizeSequence(anim, init.GetDamageState(), StandSequences.First()));
 
 			if (IsPlayerPalette)
-				p = init.WorldRenderer.Palette(Palette + init.Get<OwnerInit>().PlayerName);
+				p = init.WorldRenderer.Palette(Palette + init.Get<OwnerInit>().InternalName);
 			else if (Palette != null)
 				p = init.WorldRenderer.Palette(Palette);
 

